src/vm/callingconvention.h

```diff
@@ -611,7 +611,7 @@ class ArgIteratorTemplate : public ARGITERATOR_BASE
         if (TransitionBlock::IsFloatArgumentRegisterOffset(argOffset))
         {
             // Dividing by 8 as size of each register in FloatArgumentRegisters is 8 bytes.
-            pLoc->m_idxFloatReg = (argOffset - TransitionBlock::GetOffsetOfFloatArgumentRegisters()) / 8;
```

Comment:
@shahid-pk Oh, thanks for spotting it. Since I've already merged it in, I'll fix it in a separate change.